pkg/agent/datamodel/sig_config.go

```diff
@@ -691,6 +691,7 @@ func withEdgeZoneConfig(acsSigConfig SIGConfig) SigImageConfigOpt {
 	}
 }
 
+//nolint:unparam,nolintlint
```

Comment:
this is odd...

do you know why this one fails? does it not capture the closure somehow?

Reply:
Made this more clear